pkgs/top-level/all-packages.nix

```diff
@@ -15826,12 +15826,6 @@ in
 
   bluez = bluez5;
 
-  bluezFull = bluez.override {
-    enableWiimote = true;
-    enableMidi = true;
-    enableSixaxis = true;
-  };
-
   inherit (python3Packages) bedup;
 
   bolt = callPackage ../os-specific/linux/bolt { };
```

Comment:
Can you fix eval? There are still some `bluez5.overrides`:

https://github.com/NixOS/nixpkgs/blob/f13d9f2704cdd5ee49cf80f6514f5baf9eb9f2ac/pkgs/top-level/all-packages.nix#L25280-25284